classes/image_types_ostree.bbclass

```diff
@@ -194,23 +194,20 @@ IMAGE_CMD_garagesign () {
             exit 1
         fi
 
-	if [ ! -d "${GARAGE_SIGN_REPO}" ]; then
-            garage-sign init --repo ${GARAGE_SIGN_REPO} --home-dir ${GARAGE_SIGN_REPO} --credentials ${SOTA_PACKED_CREDENTIALS}
-        fi
-
-        reposerver_args="--reposerver $( unzip -p ${SOTA_PACKED_CREDENTIALS} tufrepo.url )"
+        rm -rf ${GARAGE_SIGN_REPO}
+        garage-sign init --repo ${GARAGE_SIGN_REPO} --home-dir ${GARAGE_SIGN_REPO} --credentials ${SOTA_PACKED_CREDENTIALS} --no-auth
```

Comment:
I think using `--no-auth` is causing garage-sign to fail for me with `The supplied authentication is not authorized to access this resource`.

Reply:
Now I understand that `--no-auth` is necessary for Spekulatius but I think that breaks everything else.